vNext/UnityGLTFv2.cs

```diff
@@ -66,7 +73,7 @@ public class UnityGLTFImporterConfig
 /// </summary>
 public class IDataLoader
 {
-    Task<Stream> LoadStream(string uri);
+    Task<Stream> LoadStreamAsync(string uri, CancellationToken ct = CancellationToken.None);

```

Comment:
How about `Uri uri` instead of `string uri`. Just require at the contract level that a valid Uri is passed in.

Reply:
Because it's not necessarily a web uri, which seems to be what the uri class is built around. Maybe path is a better parameter name?

Reply:
The [URI documentation](https://docs.microsoft.com/en-us/dotnet/api/system.uri?view=netframework-4.7.2) says that it supports the `file://` protocol, which in turn supports relative URIs. Seems like that would be enough.